gazebo/rendering/ScenePrivate.hh

```diff
@@ -145,6 +148,9 @@ namespace gazebo
       public: std::vector<OculusCameraPtr> oculusCameras;
 #endif
 
+      /// \brief The ogre scene manager factory.
+      public: CustomSceneManagerFactory *managerFactory = nullptr;
```

Comment:
I think this line needs the version macros

Reply:
fixed